src/components/App.js

```diff
@@ -6,13 +6,16 @@ import Input from 'components/Input/Input'
 import TodoList from 'components/TodoList/TodoList'
 import Todo from 'components/Todo/Todo'
 import Button from 'components/Button/Button'
+import Modal from 'components/Modal/Modal'
 import styles from 'components/app.module.scss'
 
 import todosList from 'data'
 
 const App = () => {
   const [todos, setTodos] = useState(todosList)
   const [input, setInput] = useState('')
+  const [isModalShow, setIsModalShow] = useState(false)
```

Comment:
Es una pavada, pero ya que están con demás correcciones, en inglés sería `shown` o `showed` (presente perfecto). También pueden usar `visible`.

```suggestion
  const [isModalShown, setIsModalShown] = useState(false)
```